src/PrestaShopBundle/Translation/TranslatorComponent.php

```diff
@@ -29,6 +29,9 @@
 
 use Symfony\Component\Translation\Translator as BaseTranslatorComponent;
 
+/**
+ * Translator used by Context
```

Comment:
If I'm not mistaken now it's only used in FO right?

```suggestion
 * Translator used in FO
```